src/main/java/uk/nhs/digital/nhsconnect/nhais/mesh/MeshWorkflowUnknownException.java

```diff
@@ -0,0 +1,9 @@
+package uk.nhs.digital.nhsconnect.nhais.mesh;
+
+import uk.nhs.digital.nhsconnect.nhais.exceptions.NhaisBaseException;
+
+public class MeshWorkflowUnknownException extends NhaisBaseException {
+    public MeshWorkflowUnknownException(String message) {
```

Comment:
`UnknownMeshWorkflowException` ?

Reply:
Thrown when downloaded MESH message has different workflowId than we support